src/browserClip.ts

```diff
@@ -151,8 +151,27 @@ module.exports = (params: ClipParams) => {
 
   let effects = [];
 
+  function createEffect(eff: any) {
+    if (typeof eff === 'string') {
+      return new Tone[eff]();
+    } else {
+      return eff;
+    }
+  }
+
+  function startEffect(eff: any) {
+    if (typeof eff.start === 'function') {
+      return eff.start();
```

Comment:
Is `start()` a new param in v14? Was looking at this page from 13.x as an example and couldnt find a `start` method https://tonejs.github.io/docs/13.8.25/AutoWah

Reply:
Hmmm...  🤔 No, it's not new. `AutoWah` doesn't have this method. It seems that the `start()` method is present for some effects only. E.g:
- for `AutoFilter` the method is present [in v13](https://tonejs.github.io/docs/13.8.25/AutoFilter#start) and [in v14](https://tonejs.github.io/docs/14.7.11/AutoFilter#start)
- for `AutoWah` the method is absent for both versions.

Your comment reminded me that I tested only with Tone v14, but not with with Tone v13. So I did some tests with Tone v13 and it seems ok.